samwise.js

```diff
@@ -3,28 +3,31 @@
  *
```

Comment:
SCRIPT-You did a great job handling all the possible error states in the UI. It enhances the user experience significantly.